phpBB/phpbb/db/migration/data/v33x/add_user_last_active.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+use phpbb\db\migration\migration;
+
+class add_user_last_active extends migration
+{
+	public static function depends_on()
+	{
+		return [
+			'\phpbb\db\migration\data\v33x\v3311',
+		];
+	}
+
+	public function update_schema()
+	{
+		return [
+			'add_columns'	=> [
+				$this->table_prefix . 'users'	=> [
+					'user_last_active'		=> ['TIMESTAMP', 0, 'after' => 'user_lastvisit'],
```

Comment:
I think this column can also use an index for query optimization.

Reply:
For which query?

Reply:
For the usages in variables `$sort_key_sql['l']` and `$sql_where` in `memberlist.php`

Reply:
An index doesn't actually add any improvement to the query as far as I can see. There is also no index for `user_lastvisit`.

Reply:
Oh weird, thanks for checking.